packages/bot/src/scratch/hooks/toolbox.js

```diff
@@ -11,7 +11,7 @@ import { translate }  from '../../utils/lang/i18n';
  * This is the sum of the width of the flyout (250) and some extra padding (25).
  * @type {number}
  */
-Blockly.Toolbox.prototype.width = 275;
+Blockly.Toolbox.prototype.width = 0;
```

Comment:
Should we at least add some padding e.g. 25 like before. Also the comment on top is no longer related, could be updated.